src/Analyzer.Core.UnitTests/templates/DoubleNestedFail.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "innerResourceGroup": {
+      "type": "string"
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    }
+  },
+  "resources": [
+
```

Comment:
remove empty line here

Reply:
I added some 'random' whitespaces in some templates to test whether line number mapping would still work if a user template was not properly formatted.

Reply:
Can you add comments in the templates to explain why the whitespaces?